packages/react-router-dom/modules/index.js

```diff
@@ -7,6 +7,7 @@ export Prompt from "./Prompt";
 export Redirect from "./Redirect";
 export Route from "./Route";
 export Router from "./Router";
+import RouterContext from "./RouterContext";
```

Comment:
@timdorr are you mean `export RouterContext from "./RouterContext";`?

Reply:
Wow, I'm dumb. Thanks!

Reply:
@timdorr Can you update `next` version with this fix and how I can help to speed up migration to the new context?

P.S We can't test `next` version without export its context

Reply:
I pushed out 4.4.0-alpha.1 with this fix now.